octobot_trading/api/portfolio.py

```diff
@@ -57,3 +58,15 @@ async def refresh_real_trader_portfolio(exchange_manager) -> bool:
     return await exchange_channel.get_chan(octobot_trading.constants.BALANCE_CHANNEL, exchange_manager.id). \
         get_internal_producer(). \
         refresh_real_trader_portfolio(True)
+
+
+def get_portfolio_amounts(portfolio,
+                          portfolio_type=octobot_commons.constants.PORTFOLIO_TOTAL):
```

Comment:
We are not using PORTFOLIO_TOTAL anymore, I don't understand this method

Reply:
It's not used anymore, I'm removing it